receiver/podmanreceiver/receiver_test.go

```diff
@@ -78,21 +85,47 @@ func TestScraperLoop(t *testing.T) {
 			Error: "",
 		}
 	}()
+	err = r.Start(context.Background(), componenttest.NewNopHost())
+	require.NoError(t, err)
+	md := <-consumerForMetrics
+	assert.Equal(t, md.ResourceMetrics().Len(), 1)
 
-	r.Start(context.Background(), componenttest.NewNopHost())
+	err = r.Shutdown(context.Background())
+	require.NoError(t, err)
+}
 
-	md := <-consumer
-	assert.Equal(t, md.ResourceMetrics().Len(), 1)
+func TestLogsLoop(t *testing.T) {
```

Comment:
Could you add also a test that exercises getting two events based on the streaming connection with podman?

Reply:
Added